webapp/channels/src/components/channel_settings_modal/channel_settings_info_tab.tsx

```diff
@@ -106,10 +106,10 @@ function ChannelSettingsInfoTab({
     useEffect(() => {
         // Calculate unsaved changes directly
         const unsavedChanges = channel ? (
-            displayName !== channel.display_name ||
-            channelUrl !== channel.name ||
-            channelPurpose !== channel.purpose ||
-            channelHeader !== channel.header ||
+            displayName.trim() !== channel.display_name ||
```

Comment:
For all changed files-

It will be better if we saved the trimmed value in state rather then remembering to always trim the value before using it in all the places. This will minimize the required changes, and eliminate the error when forgets to trim the value before using it (which will definitely happen at some point)